### PR TITLE
ImprovMX Record

### PR DESCRIPTION
**Pull request by xzonix**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

fran.is-a.dev

### CODE REVIEW

File: domains/mail.fran.json
Comment (Stef-00012) [inline domains/mail.fran.json:1]: Missing starting `{`
Missing `owner` field
records should be inside the `record` field

File: domains/mail.fran.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ]
+    ],
```
Comment (Stef-00012) [inline domains/mail.fran.json:5]: Remove this `]`

File: domains/mail.fran.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
+  }
```
Comment (Stef-00012) [inline domains/mail.fran.json:7]: remove this `}`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/mail.fran.json
Comment (xzonix) [reply domains/mail.fran.json:1]: can you like tell me step by step what should i add to the JSON to work? i'm not gonna lie i don't know what are you trying to say, sorry for my stupidity.

**Pull request closed without merging**